views/templates/books-index.html

```diff
@@ -0,0 +1,87 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1">
+
+      <form class="form-horizontal" ng-submit="booksIndexCtrl.createBook()">
+        <fieldset>
+
+          <!-- Form Name -->
+          <legend>Add New Book</legend>
+
+          <!-- Text input for book title-->
+          <div class="form-group">
+            <label class="col-md-4 control-label">Book Title</label>
+            <div class="col-md-4">
+            <input id="title" ng-model="booksIndexCtrl.newBook.title" type="text" class="form-control input-md" required="">
+            </div>
+          </div>
+
+          <!-- Text input book author-->
+          <div class="form-group">
+            <label class="col-md-4 control-label">Author Name</label>
+            <div class="col-md-4">
+            <input ng-model="booksIndexCtrl.newBook.author" type="text" class="form-control input-md">
+            </div>
+          </div>
+
+          <!-- Text input for image url-->
+          <div class="form-group">
+            <label class="col-md-4 control-label">Image Url:</label>
+            <div class="col-md-4">
+            <input ng-model="booksIndexCtrl.newBook.image" type="text" class="form-control input-md">
```

Comment:
indentation; this is a child of the div above it so should be more indented. Or, you might not even need that div and could just put the col-md-4 class on this input.